src/mgmt/srv_pool.c

```diff
@@ -980,3 +980,38 @@ ds_mgmt_pool_delete_acl(uuid_t pool_uuid, const char *principal,
 out:
 	return rc;
 }
+
+int
+ds_mgmt_pool_set_prop(uuid_t pool_uuid, daos_prop_t *prop)
```

Comment:
ok, i'll take a look at this.

Reply:
Looking at this, I'm not sure it makes sense. ds_mgmt_pool_overwrite_acl() still needs the svc/ranks stuff in order to call get_acl_for_pool() at the bottom, so it doesn't seem like it would really save anything unless I'm misunderstanding your suggestion.

Reply:
Ah, good point. Let's just leave it as-is.